com.unity.perception/Editor/AssetPreparation/AssetPreparationTools.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.IO;
+using UnityEngine;
+
+namespace UnityEditor.Perception.AssetPreparation
+{
+    public class CreatePrefabFromSelectedAssets : MonoBehaviour
```

Comment:
I think this class is defined as a Monobehaviour that can actually be added to a GameObject. I think you can hide this MonoBehaviour from the add component menu by making the class internal while still exposing the MenuItem to the editor.